src/Components/IndexScene/IndexScene.tsx

```diff
@@ -304,6 +320,60 @@ export class IndexScene extends SceneObjectBase<IndexSceneState> {
     return getContentScene(this.state.routeMatch?.params.breakdownLabel);
   }
 
+  private provideAssistantContext() {
+    const setAssistantContext = providePageContext(`${PLUGIN_BASE_URL}/**`, []);
+    const updateAssistantContext = () => {
+      const contexts = [];
+
+      const dsVar = getDataSourceVariable(this);
+      const ds = getDataSourceSrv()
+        .getList({
+          type: 'loki',
+        })
+        .find((ds) => ds.uid === dsVar.state.value.toString());
```

Comment:
There is a getLokiDatasource() function

Reply:
Changed in 1ae2f32e36c545ba1309607c3a719cc8d6a31e57